source/_drafts/Video-js-6-0-0-RC-0-The-first-RC.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Video.js 6.0.0-RC.0: The first Release Candidate'
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+  - video.js 6.0
+---
+
+## Release Candiate Released
```

Comment:
What about something like: "The first release candidate for 6.0 has been release"?